core/src/com/mygdx/pirategame/Coin.java

```diff
@@ -110,4 +113,25 @@ public void draw(Batch batch) {
             super.draw(batch);
         }
     }
+
+    @Test
+    public void testCoinUpdate() {
+        Coin testCoin = new Coin(new GameScreen(new PirateGame()), 10,10);
+
+        // Schedule a destroy
+        testCoin.setToDestroyed = true;
```

Comment:
Discrepancy here - this file uses `setToDestroyed`, others use `setToDestroy`

Not our fault or a major issue - just a bit annoying to look at!